apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -67,6 +68,10 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
     const [ profileSchema, setProfileSchema ] = useState<ProfileSchema[]>();
     const [ isEmailPending, setEmailPending ] = useState<boolean>(false);
     const [ showEditAvatarModal, setShowEditAvatarModal ] = useState<boolean>(false);
+    const [ showMobileUpdateWizard, setShowMobileUpdateWizard ] = useState<boolean>(false);
+
+    // TODO: Get from config
+    const isMobileVerificationEnabled: boolean = true;
```

Comment:
Let's get this from `deployment.config.json` within the app for time being.

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70